proto/pulumi/provider.proto

```diff
@@ -191,9 +192,10 @@ message CheckFailure {
 message DiffRequest {
     string id = 1;                     // the ID of the resource to diff.
     string urn = 2;                    // the Pulumi URN for this resource.
-    google.protobuf.Struct olds = 3;   // the old values of provider inputs to diff.
-    google.protobuf.Struct news = 4;   // the new values of provider inputs to diff.
+    google.protobuf.Struct olds = 3;   // the old output values of resource to diff.
```

Comment:
Hm, perhaps it's just a vocab confusion between TF and Pulumi vocabulary. Perhaps the Pulumi vocabulary "old output values" essentially translates to "prior state" in the TF world, since TF bridge always publishes as Pulumi "outputs" all properties; the schema for output properties always includes every input property. Yeah this is probably fine.

Reply:
Right, right. So in the Pulumi language it's properly called "outputs". It's good then.